cdci_data_analysis/analysis/queries.py

```diff
@@ -246,9 +246,13 @@ def get_parameters_list_as_json(self,prod_dict=None):
         l=[ {'query_name':self.name}]
 
         for par in self._parameters_list:
-            l.append(par.reprJSON())
-
-        return json.dumps(l)
+            l.extend(par.reprJSON())
+        # remove duplicates like T_format
+        seen = set()
+        l1 = [x for x in l if (x.get('name') is None) or not (x.get('name') in seen or seen.add(x.get('name')))]
```

Comment:
I am not sure this is the most readable way to do it. It's not like the goal is to minimize the number of lines. Also the line is a bit too long.